noxfile.py

```diff
@@ -31,45 +35,32 @@
 
 PYTHON_DEFAULT_VERSION = PYTHON_VERSIONS[-1]
 
-PY_PATHS = ['b2sdk', 'test', 'noxfile.py', 'setup.py']
-
-REQUIREMENTS_FORMAT = ['yapf==0.27', 'ruff==0.0.270']
-REQUIREMENTS_LINT = REQUIREMENTS_FORMAT + ['pytest==6.2.5', 'liccheck==0.6.2']
-REQUIREMENTS_RELEASE = ['towncrier==23.11.0']
-REQUIREMENTS_TEST = [
-    "pytest==6.2.5",
-    "pytest-cov==3.0.0",
-    "pytest-mock==3.6.1",
-    'pytest-lazy-fixture==0.6.3',
-    'pytest-xdist==2.5.0',
-    'pytest-timeout==2.1.0',
-]
-REQUIREMENTS_BUILD = ['setuptools>=20.2', 'wheel>=0.40']
+PY_PATHS = ['b2sdk', 'test', 'noxfile.py']
 
 nox.options.reuse_existing_virtualenvs = True
 nox.options.sessions = [
     'lint',
     'test',
 ]
 
-# In CI, use Python interpreter provided by GitHub Actions
-if CI:
-    nox.options.force_venv_backend = 'none'
 
+def pdm_install(session: nox.Session, *groups: str, dev=True) -> None:
```

Comment:
nitpick: not sure why name change, as this make this signature more implementation specific; but then again it doesn't matter much